Power Grid Model Benchmark.ipynb

```diff
@@ -1348,96 +10241,96 @@
+       "      <td id=\"T_727bf_row0_col3\" class=\"data row0 col3\" >20.63x</td>\n",
+       "      <td id=\"T_727bf_row0_col4\" class=\"data row0 col4\" >1.00x</td>\n",
+       "      <td id=\"T_727bf_row0_col5\" class=\"data row0 col5\" >0.00x</td>\n",
        "    </tr>\n",
        "    <tr>\n",
-       "      <th id=\"T_2c730_level0_row1\" class=\"row_heading level0 row1\" >Symmetric calculation without solver initialization</th>\n",
-       "      <td id=\"T_2c730_row1_col0\" class=\"data row1 col0\" >68.75x</td>\n",
-       "      <td id=\"T_2c730_row1_col1\" class=\"data row1 col1\" >46.12x</td>\n",
-       "      <td id=\"T_2c730_row1_col2\" class=\"data row1 col2\" >23.22x</td>\n",
-       "      <td id=\"T_2c730_row1_col3\" class=\"data row1 col3\" >17.65x</td>\n",
-       "      <td id=\"T_2c730_row1_col4\" class=\"data row1 col4\" >1.00x</td>\n",
-       "      <td id=\"T_2c730_row1_col5\" class=\"data row1 col5\" >0.00x</td>\n",
+       "      <th id=\"T_727bf_level0_row1\" class=\"row_heading level0 row1\" >Symmetric calculation without solver initialization</th>\n",
+       "      <td id=\"T_727bf_row1_col0\" class=\"data row1 col0\" >45.41x</td>\n",
+       "      <td id=\"T_727bf_row1_col1\" class=\"data row1 col1\" >43.75x</td>\n",
+       "      <td id=\"T_727bf_row1_col2\" class=\"data row1 col2\" >24.25x</td>\n",
+       "      <td id=\"T_727bf_row1_col3\" class=\"data row1 col3\" >32.42x</td>\n",
+       "      <td id=\"T_727bf_row1_col4\" class=\"data row1 col4\" >1.00x</td>\n",
+       "      <td id=\"T_727bf_row1_col5\" class=\"data row1 col5\" >0.00x</td>\n",
        "    </tr>\n",
        "    <tr>\n",
-       "      <th id=\"T_2c730_level0_row2\" class=\"row_heading level0 row2\" >Asymmetric calculation with solver initialization</th>\n",
-       "      <td id=\"T_2c730_row2_col0\" class=\"data row2 col0\" >300.83x</td>\n",
-       "      <td id=\"T_2c730_row2_col1\" class=\"data row2 col1\" >426.60x</td>\n",
-       "      <td id=\"T_2c730_row2_col2\" class=\"data row2 col2\" >211.51x</td>\n",
-       "      <td id=\"T_2c730_row2_col3\" class=\"data row2 col3\" >419.35x</td>\n",
-       "      <td id=\"T_2c730_row2_col4\" class=\"data row2 col4\" >1.00x</td>\n",
-       "      <td id=\"T_2c730_row2_col5\" class=\"data row2 col5\" >32.77x</td>\n",
+       "      <th id=\"T_727bf_level0_row2\" class=\"row_heading level0 row2\" >Asymmetric calculation with solver initialization</th>\n",
+       "      <td id=\"T_727bf_row2_col0\" class=\"data row2 col0\" >599.37x</td>\n",
+       "      <td id=\"T_727bf_row2_col1\" class=\"data row2 col1\" >669.71x</td>\n",
+       "      <td id=\"T_727bf_row2_col2\" class=\"data row2 col2\" >250.72x</td>\n",
+       "      <td id=\"T_727bf_row2_col3\" class=\"data row2 col3\" >479.83x</td>\n",
+       "      <td id=\"T_727bf_row2_col4\" class=\"data row2 col4\" >1.00x</td>\n",
+       "      <td id=\"T_727bf_row2_col5\" class=\"data row2 col5\" >37.17x</td>\n",
```

Comment:
seems like the slowing is caused by PP regressions, but we are benefiting much more than OpenDSS